app/views/auth/registrations/_sessions.html.haml

```diff
@@ -19,10 +19,10 @@
         %td
           %samp= session.ip
         %td
-          - if request.session['auth_id'] == session.session_id
+          - if current_session.id == session.session_id
```

Comment:
not working.

for example, `current_session.id` is 34, but `session.session_id` is `816a3488dfacc7912615de4977ccdd2a`.